### PR TITLE
Fix fees for accounts with low balances

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

The fee estimation was using 1 dollar (instead of the intended 1 wei) to calculate fees, which would raise an error that was not caught.
This fixes the fee amount and adds error handling.
We should improve this further, see #585 

### Tested

did Send flow on account with balance of 3 cents.

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/src/fees/saga.ts [hunk 1]
```diff
@@ -1,51 +1,60 @@
...
+    switch (feeType) {
+      case FeeType.INVITE:
+        feeInWei = yield call(getInvitationVerificationFee)
+        break
+      case FeeType.SEND:
+        // Just use default values here since it doesn't matter for fee estimation
+        feeInWei = yield call(getSendFee, account, getStableTokenContract, {
+          recipientAddress: CeloDefaultRecipient.address,
+          amount: web3.utils.fromWei('1'),
+          comment: 'Coffee or Tea?',
```
Comment (martinvol) [inline packages/mobile/src/fees/saga.ts:35]: why aren't we using the actual comment and amount the user is meaning to send? We have an edge case here where an extra character can make you run out of money to pay for gast.

Comment (jmrossy) [reply packages/mobile/src/fees/saga.ts:35]: Because this fee estimation is done before the user inputs in amount and/or comment

Comment (marekolszewski) [reply packages/mobile/src/fees/saga.ts:35]: Maybe we should include something that is the max permissible comment length here?

File: packages/mobile/src/fees/saga.ts [hunk 2]
```diff
@@ -1,51 +1,60 @@
...
+
+    switch (feeType) {
+      case FeeType.INVITE:
+        feeInWei = yield call(getInvitationVerificationFee)
+        break
+      case FeeType.SEND:
+        // Just use default values here since it doesn't matter for fee estimation
+        feeInWei = yield call(getSendFee, account, getStableTokenContract, {
+          recipientAddress: CeloDefaultRecipient.address,
+          amount: web3.utils.fromWei('1'),
```
Comment (marekolszewski) [inline packages/mobile/src/fees/saga.ts:34]: 😄

**Review by marekolszewski — APPROVED** [review]

**Pull request merged**